components/mpas-albany-landice/bld/namelist_files/albany_input.yaml

```diff
@@ -22,13 +40,21 @@ ANONYMOUS:
         Combo Type: OR
         Number of Tests: 2
         Test 0: 
-          Test Type: NormF
-          Norm Type: Two Norm
-          Scale Type: Scaled
-          Tolerance: 1.0e-05
+          Test Type: Combo
+          Combo Type: AND
+          Number of Tests: 2
+          Test 0: 
+            Test Type: NormF
+            Norm Type: Two Norm
+            Scale Type: Unscaled
+            Tolerance: 1.0e-03
+          Test 1:
+            Test Type: RelativeNormF
+            Norm Type: Two Norm
+            Tolerance: 0.9999
         Test 1: 
           Test Type: MaxIters
-          Maximum Iterations: 50
+          Maximum Iterations: 100
```

Comment:
Do we definitely want to raise the max iterations this high?  In my experience, if you hit 50, you're not going to converge anyway, and you waste a lot of time with a larger number.  But maybe you've identified this is needed for this mesh?

Reply:
This has actually made a difference. I agree that this could be problematic in the long term for the reasons you note, but right now, this change has at least allowed the new 1-to-10km init. cond. to run successfully in E3SM for several time steps (I've not yet tested / confirmed exactly how many time steps can be run).

Reply:
100 iterations is not that bad (especially for the initial solve) because we are now using adaptive tolerance of the linear solvers, so the linear solvers only take a few iterations per nonlinear solver iteration.

Reply:
Ok, thanks for confirming that